src/neuroglancer/datasource/precomputed/frontend.ts

```diff
@@ -124,6 +125,7 @@ class ScaleInfo {
           x => parseFixedLengthArray(vec3.create(), x, verifyPositiveInt));
     }
     this.key = verifyObjectProperty(obj, 'key', verifyString);
+    this.hidden =  verifyObjectProperty(obj, 'hidden', verifyOptionalBoolean);
```

Comment:
add `?? false` to the end so that `this.hidden` has type `boolean` rather than `boolean | undefined`